vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/steps/integration/SSLLabsSteps.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.steps.integration;
+
+import java.util.Optional;
+
+import org.jbehave.core.annotations.Then;
+import org.vividus.softassert.ISoftAssert;
+import org.vividus.ssllabs.Grade;
+import org.vividus.ssllabs.SSLLabsClient;
+import org.vividus.steps.ComparisonRule;
+
+public class SSLLabsSteps
+{
+    private final SSLLabsClient sslLabsClient;
+    private final ISoftAssert softAssert;
+
+    public SSLLabsSteps(SSLLabsClient sslLabsClient, ISoftAssert softAssert)
+    {
+        this.sslLabsClient = sslLabsClient;
+        this.softAssert = softAssert;
+    }
+
+    /**
+     * Performs SSL scanning using www.ssllabs.com and compares received grade value with expected one.
+     * @param url The URL for SSL scanning and grading.
+     * @param comparisonRule The rule to compare values. The supported rules:
+     *                       <ul>
+     *                       <li>less than (&lt;)</li>
+     *                       <li>less than or equal to (&lt;=)</li>
+     *                       <li>greater than (&gt;)</li>
+     *                       <li>greater than or equal to (&gt;=)</li>
+     *                       <li>equal to (=)</li>
+     *                       <li>not equal to (!=)</li>
+     *                       </ul>
+     * @param gradeName The name of grade
+     * (<i>The possible values:<b>'A+','A','A-','B','C','D','E','F','T','M'</b></i>).
+     */
+    @Then("SSL rating for URL `$url` is $comparisonRule `$gradeName`")
+    public void compareGrades(String url, ComparisonRule comparisonRule, String gradeName)
```

Comment:
```suggestion
    public void compareGrades(String url, ComparisonRule comparisonRule, Grade gradeName)
```

and rename `Grade#getGradeByName ` -> `Grade#fromString`